dev-util/meson/meson-0.58.2-r1.ebuild

```diff
@@ -1,16 +1,15 @@
-# Copyright 2016-2020 Gentoo Authors
+# Copyright 2016-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
-PYTHON_COMPAT=( python3_{6,7} )
-DISTUTILS_USE_SETUPTOOLS="rdepend"
+PYTHON_COMPAT=( python3_{5,6,7} )
```

Comment:
Same here. Any reason for bumping meson actually? `systemd` requires the new version?

Reply:
The upstream uses meson-multilib which requires a newer version of meson, and the meson pulls meson-format-array.